docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,381 @@
+
+ private_key_file = %%install_dir%%/conf/ssl/server.key
+ certificate_file = %%install_dir%%/conf/ssl/server.pem
+
+E.g.
+
+ private_key_file = %%install_dir%%/conf/ssl/tls_certs/YourCert.key
+ certificate_file = %%install_dir%%/conf/ssl/tls_certs/YourCert.pem
+ CA_file = %%install_dir%%/conf/ssl/tls_certs/MyCA.pem
+
+Certificate revocation checks also have to be configured using OCSP in the same block.
+
+For example: 
+
+    ocsp {
+        enable = yes
+        override_cert_url = yes
+        url = "http://<ServerDNSName or IP>/ocsp"
+    }
+
+Restart radiusd to regenerate the new configuration files and enable EAP-TLS using your CA signed certificates:
+
+ # /usr/local/pf/bin/pfcmd service radiusd restart
+
+PacketFence provider configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the PKI requires configuring the PKI providers section in the PacketFence GUI under Configuration > Users. The provider configuration defines how PacketFence connects to the MS PKI and what information will be send.
+
+Add a new PKI provider and select MS PKI.
+
+Fill out the form for a PKI provider according to your Certificate of Authority configuration.
+
+image::docs/images/scep-ms-pki-provider.png[scaledwidth="100%",alt="MSPKI configuration"]
+
+For the url it will be http://<ServerDNSName>/CertSrv/mscep if you use our configuration.
+
+image::docs/images/scep-ms-pki-provider2.png[scaledwidth="100%",alt="MSPKI configuration"]
+
+The "server certificate path" and "CA cert path" both need to be absolute (e.g. /usr/local/pf/conf/ssl/tls_certs/MyCA.pem is an absolute path).
+
+The "Common name attribute" field defines how the certificate will be generated and what type of "ownership" will associate the certificate to the connection.
+If you select 'MAC address', a certificate will be generated for the device itself using the MAC address as the identifier.
+If you select 'Username', a certificate will be generated for the user using his login name on the authentication backend (e.g. Active-Directory).
+
+This means that revoking the certificate for a username based certificate will block all the devices that this user registered.
+If you generate the certificates using the MAC address, revoking a certificate will block only that device.
+
+Provisioners configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Provisioners allow devices to automatically configure themselves to connect to the proper SSID (if applicable), use the proper authentication method (e.g. EAP-TLS) and trust the CA certificate and any certificate signed by it.
+
+Provisioners are configured in the PacketFence administration GUI under Configuration > Users > Provisioners.
+
+Add a new provisioner for each of the classes of devices to be supported amongst Android, Apple Devices and Windows.
+Fill out the form, choosing a different Provisioning Id per provisioner.
+
+The fields affect the provisioning behavior in the following fashion:
```

Comment:
Your fields have make-up and nice clothes ?
What is a field fashion ?
Field Summer collection 2015